Makefile

```diff
@@ -78,17 +81,25 @@ bdist: npm-packages
 
 install: bdist lint ## Build distribution and install
 	pip install --upgrade dist/elyra-*-py3-none-any.whl
-	$(call UNLINK_LAB_EXTENSION,application)
-	$(call UNLINK_LAB_EXTENSION,theme-extension)
-	$(call UNLINK_LAB_EXTENSION,code-snippet-extension-experimental)
-	$(call UNLINK_LAB_EXTENSION,pipeline-editor-extension)
-	$(call UNLINK_LAB_EXTENSION,python-runner-extension)
-	jupyter lab clean
 	$(call LINK_LAB_EXTENSION,application)
-	$(call LINK_LAB_EXTENSION,theme)
-	$(call LINK_LAB_EXTENSION,code-snippet)
-	$(call LINK_LAB_EXTENSION,pipeline-editor)
-	$(call LINK_LAB_EXTENSION,python-runner)
+	jupyter lab build
+	jupyter serverextension list
+	jupyter labextension list
+
+install-backend: ## Build and install backend
```

Comment:
can we call this `install-server` to be consistent with `test-server`? The other way around is also ok.

Reply:
Sure, I like that better as well. (This was an already existing target that I just reused so I didn't want to edit it)